README.md

```diff
@@ -16,7 +16,7 @@ Folder `api` contains source code of the API implementation. This repository rel
 
 ## Requirements.
 
-Python == 3.11
+Python <= 3.10
```

Comment:
It would be good to explain in the file why we need to limit the Python version.

Reply:
Documentation added!